app/controllers/application_controller.rb

```diff
@@ -1,2 +1,11 @@
 class ApplicationController < ActionController::API
+	include DeviseTokenAuth::Concerns::SetUserByToken
+	before_action :modify_permitted_params, if: :devise_controller?
+
+	protected
+
+	def modify_permitted_params
+		added_attributes = [:name]
+		devise_parameter_sanitizer.permit(:sign_up, keys: added_attributes)
+	end
 end
```

Comment:
fix the indentation in this file

Reply:
this appears to be indented properly already in my vscode

Reply:
change from tabs to spaces 👍🏻